_data/sidebars/k8smain-sidebar.yml

```diff
@@ -289,6 +294,11 @@ entries:
       output: web, pdf
       type: page
 
+    - title: Using the OPA Gatekeeper charm
```

Comment:
Note the comment on the card - Not sure whether people will be looking for 'gatekeeper' or 'policy management'

Reply:
For now, i think gatekeeper is appropriate.  We can cross the more generic "policy mgmt" bridge if we ever support something else.

Reply:
I think gatekeeper is fine for now.  We can cross the "policy mgmt" bridge if/when we support something besides opa.